ViewControllers/Search/SearchDataSourceController.swift

```diff
@@ -29,6 +29,12 @@ final class SearchDataSourceController: NSObject {
         self.artists = artists
         self.tableView?.reloadData()
     }
+    
+    func resetArtist() {
+        
+        self.artists = []
+        self.tableView?.reloadData()
```

Comment:
if every time you set artists table should be reloaded, you can put `reloadData` to `didSet` for artists